code/controllers/subsystem/shuttles.dm

```diff
@@ -135,8 +135,6 @@ var/datum/subsystem/shuttle/SSshuttle
 /datum/subsystem/shuttle/proc/canRecall()
 	if(emergency.mode != SHUTTLE_CALL)
 		return
-	if(ticker.mode.name == "meteor")
```

Comment:
if it's needed there's going to be better ways to do it that specific checks like this.